<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleGUI</class>
 <widget class="QMainWindow" name="SimpleGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Slicer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/ISS.ico</normaloff>:/ISS.ico</iconset>
  </property>
  <widget class="QWidget" name="mainWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="innerWidget">
      <item>
       <widget class="QSplitter" name="class_one_splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="opaqueResize">
         <bool>true</bool>
        </property>
        <property name="childrenCollapsible">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_5">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QComboBox" name="class1ComboBox"/>
          </item>
          <item>
           <widget class="QListWidget" name="class1List">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="imagesViewHorizontalLayout">
            <item>
             <layout class="QVBoxLayout" name="rightImageLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="slice1Label">
                  <property name="text">
                   <string>Slice &amp;1 ID</string>
                  </property>
                  <property name="buddy">
                   <cstring>id1</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="id1">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>75</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="maximum">
                   <number>999999999</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="histogram1">
                  <property name="text">
                   <string>Histogram 1</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="slice1View">
                  <property name="maximumSize">
                   <size>
                    <width>300</width>
                    <height>300</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Slice 1 View</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="leftImageLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QSpinBox" name="id2">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>75</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="maximum">
                   <number>999999999</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="slice2Label">
                  <property name="text">
                   <string>Slice &amp;2 ID</string>
                  </property>
                  <property name="buddy">
                   <cstring>id2</cstring>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="slice2View">
                  <property name="maximumSize">
                   <size>
                    <width>300</width>
                    <height>300</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Slice 2 View</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="histogram2">
                  <property name="text">
                   <string>Histogram 2</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QListWidget" name="pairsList">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>100</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QPushButton" name="keepBtn">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>75</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Keep</string>
                  </property>
                  <property name="shortcut">
                   <string>Return</string>
                  </property>
                  <property name="default">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="removeBtn">
                  <property name="text">
                   <string>Remove</string>
                  </property>
                  <property name="shortcut">
                   <string>Del</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QSpinBox" name="pairsCount">
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::NoButtons</enum>
                  </property>
                  <property name="maximum">
                   <number>999999999</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCheckBox" name="confirmSave">
                <property name="layoutDirection">
                 <enum>Qt::RightToLeft</enum>
                </property>
                <property name="text">
                 <string>Confirm for saving</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="tristate">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="sizeLbl">
                <property name="text">
                 <string>&amp;Size:</string>
                </property>
                <property name="buddy">
                 <cstring>sliceSize</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="sliceSize">
                <property name="minimumSize">
                 <size>
                  <width>75</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>1000000277</number>
                </property>
                <property name="singleStep">
                 <number>10</number>
                </property>
                <property name="value">
                 <number>224</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="name1Lbl">
                <property name="text">
                 <string>&amp;Name 1:</string>
                </property>
                <property name="textInteractionFlags">
                 <set>Qt::LinksAccessibleByMouse</set>
                </property>
                <property name="buddy">
                 <cstring>slice1NameEdit</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="slice1NameEdit"/>
              </item>
              <item>
               <widget class="QLabel" name="name2Lbl">
                <property name="text">
                 <string>Name 2:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="slice2NameEdit"/>
              </item>
              <item>
               <widget class="QPushButton" name="saveAll">
                <property name="text">
                 <string>Save All</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_4">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QComboBox" name="class2ComboBox"/>
          </item>
          <item>
           <widget class="QListWidget" name="class2List">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="baseSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Directory"/>
    <addaction name="actionSave_To"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionLoad_Directory">
   <property name="text">
    <string>Load Directory</string>
   </property>
  </action>
  <action name="actionSave_To">
   <property name="text">
    <string>Save To</string>
   </property>
  </action>
  <action name="actionImage_1_next">
   <property name="text">
    <string>Image 1 ↑</string>
   </property>
  </action>
  <action name="actionImage_2_next">
   <property name="text">
    <string>Image 2 ↑</string>
   </property>
  </action>
  <action name="actionImage_1_previous">
   <property name="text">
    <string>Image 1 ↓</string>
   </property>
  </action>
  <action name="actionImage_2_previous">
   <property name="text">
    <string>Image 2 ↓</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>class1List</tabstop>
  <tabstop>class2List</tabstop>
  <tabstop>pairsList</tabstop>
  <tabstop>keepBtn</tabstop>
  <tabstop>removeBtn</tabstop>
  <tabstop>confirmSave</tabstop>
  <tabstop>sliceSize</tabstop>
  <tabstop>slice1NameEdit</tabstop>
  <tabstop>saveAll</tabstop>
  <tabstop>id1</tabstop>
  <tabstop>id2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
